reference/docs-conceptual/community/contributing/product-terminology.md

```diff
@@ -111,7 +111,7 @@ Guidelines
 
 - First mention - use the full name
 - Subsequent mentions - you can use "VS Code"
-- Never use "VSCode"
+- Never use "VS Code"
```

Comment:
Revert this change. This is documenting the negative case.

Reply:
Understood. I've reverted the change accordingly, since it's intended to document the negative case. Thank you for pointing that out.